packages/paste-core/components/menu/__tests__/customization.spec.tsx

```diff
@@ -0,0 +1,464 @@
+import * as React from 'react';
+import {render, screen} from '@testing-library/react';
+import {matchers} from 'jest-emotion';
+import type {RenderOptions} from '@testing-library/react';
+import {CustomizationProvider} from '@twilio-paste/customization';
+
+import {initStyles, BaseMenu} from '../stories/customization.stories';
+
+expect.extend(matchers);
+
+const PASTE_ELEMENT = 'data-paste-element';
+
+const initWrapper = (element: string): RenderOptions['wrapper'] => ({children}) => (
```

Comment:
What is `RenderOptions`? I can't find it in testing library docs unless I'm just missing it 👀